Modules/Core/Common/include/itkLightObject.h

```diff
@@ -27,6 +27,11 @@
 #include <iostream>
 #include <typeinfo>
 
+#if defined(_WIN32)
+// To get LONG defined
```

Comment:
Where is this used?

Reply:
that's a partial undo of 0dd6b51. CI was red before adding it back. So I guess the header is needed. But the comment is wrong, there's no `LONG` used in itkLightObject.h.  But I don't have Windows so I'll leave it for someone else...

Reply:
Please remove the erroneous comment.

Reply:
Removed.